src/server/_logger.py

```diff
@@ -0,0 +1,94 @@
+"""Structured logger utility for creating JSON logs in Delphi pipelines."""
```

Comment:
This was done to have a separate logger for User class(to log user's queries) as we don't have `src/acquisition/` in `delphi_web_epidata` container.
Can we use app.logger for that? Yep, but if we do this, we will need to parse app logs to get information about user's queries.

Reply:
Sure, it is an exact duplicate of that `src/acquisition/covidcast/logger.py` as we have only `src/server` in `delphi_web_epidata` container so we don't have access to that existing logger. And this duplicate is for logging user's api usage inside that container.
Can we use `app.logger`? - Yep, but then we will need to parse somehow that app logs to separate logs.
Possible solutions:

1. Leave as it is (we will have duplicated files in project)
2. Add 1 line to Dockerfile to copy `src/acquisition/covidcast/logger.py` to `delphi_web_epdiata` (the most efficient solution & no duplicates)
3. Implement custom logging level and handler to collect api usage logs (will take more time than No2, not efficient, needs additional tests)

Reply:
UPD:
Need to add `COPY logger` to `devops/Dockerfile` and `dev/ docker/python/Dockerfile` because `delphi_web_python` also needs `logger.py` in order to run tests.